docs/content/en/integrations/parsers.md

```diff
@@ -311,6 +311,12 @@ That will generate three files, two of which are needed for defectdojo. Build th
 
 From: https://github.com/aquasecurity/cloudsploit . Import the JSON output.
 
+### CSAF (Common Security Advisory Format)
+
+From: <https://www.oasis-open.org/committees/csaf/> . Import the JSON output.
+
+All findings are imported for the product that the import is executed for. Product definitions from the CSAF file are ignored as mappings from CSAF to DefectDojo products aren't always existent.
```

Comment:
Why not use those that exist ("mappings from CSAF to DefectDojo products aren't always existent")?